app.rb

```diff
@@ -135,6 +107,73 @@
   redirect to("/merchant/#{merchant.public_id}")
 end
 
+def _user_and_business(country_code)
+  case country_code
+  when "USA"
+    {
+      :user => {
+        :first_name => "Bob",
+        :last_name => "Merchant",
+        :phone => "312-555-5555",
+        :dob_day => "01",
+        :dob_month => "01",
+        :dob_year => "1970",
+        :country => "USA",
+      },
+      :business => {
+        :name => "Example CO",
+        :registered_as => "limited_liability_corporation",
+        :industry => "software",
+        :phone => "312-555-5555",
+        :website => "https://example.com",
+        :description => "send money",
+        :currency => "USD",
+        :annual_volume_amount => "50,000",
+        :average_transaction_amount => "10",
+        :maximum_transaction_amount => "100",
+        :ship_physical_goods => false,
+        :street_address => "222 W Merchandise Mart Plaza",
+        :locality => "Chicago",
+        :region => "IL",
+        :postal_code => "60654",
+        :country => "USA",
+      },
+    }
+  when "GBR"
+    {
+      :user => {
+        :first_name => "Bob",
+        :last_name => "Merchant",
+        :phone => "+4403457345345",
+        :dob_day => "01",
+        :dob_month => "01",
+        :dob_year => "1970",
+        :country => "GBR",
+      },
+      :business => {
+        :name => "Example CO",
+        :registered_as => "limited_liability_corporation",
+        :industry => "software",
+        :phone => "+4403457345345",
+        :website => "https://example.com",
+        :description => "send money",
+        :currency => "GBP",
+        :annual_volume_amount => "50,000",
```

Comment:
Nitpick: does this value come from a drop-down, or is it entered in? If it's entered in, then as we expand to more countries, we may need to keep in mind that [many countries](https://en.wikipedia.org/wiki/Decimal_mark#Hindu.E2.80.93Arabic_numeral_system) use `,` as a decimal mark.